test/integration/call_list_test.rb

```diff
@@ -1,8 +1,7 @@
-require 'test_helper'
+require 'application_system_test_case'
```

Comment:
Test Helper still exists, but application_system_test_case is just for the new systemtestcase stuff. I've been trying to keep them broken apart.